config/pmd-test.xml

```diff
@@ -85,6 +86,7 @@
             | /ClassOrInterfaceDeclaration[@Image='AstRegressionTest']//MethodDeclarator[@Image='testImpossibleExceptions']
             | //ClassOrInterfaceDeclaration[@Image='AstRegressionTest']//MethodDeclarator[@Image='testImpossibleValid']
             | //ClassOrInterfaceDeclaration[@Image='AllChecksTest']//MethodDeclarator[@Image='testAllModulesAreReferencedInConfigFile']
+            | //ClassOrInterfaceDeclaration[@Image='PackageNamesLoaderTest']//MethodDeclarator[@Image='testPackagesFile']
```

Comment:
@rnveach , please inline `validatePackageNames` to testPackagesFile and this suppression will not be required.